.env.example

```
@@ -0,0 +1,13 @@
+# GitLab Configuration
+GITLAB_TOKEN=your-gitlab-token-here
+GITLAB_API_URL=http://gitlab.dimed.com.br/api/v4
+GITLAB_DEFAULT_GROUP=grupopanvel/varejo/crm
+GITLAB_BACKLOG_LABEL=Grupo Panvel :: Backlog
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITLAB_BACKLOG_LABEL key should go before the GITLAB_DEFAULT_GROUP key

(UnorderedKey)